wallet/src/database/model/WalletModel.ts

```diff
@@ -0,0 +1,51 @@
+import { DataTypes, Model, ModelOptions } from "sequelize";
+
+export class WalletModel extends Model<{
+  id?: string;
+  user_id: string;
+  rapyd_e_wallet: string;
+}> {
+  public static tableName = "wallet";
+
+  public static rawAttributes = {
+    id: {
+      type: DataTypes.UUID,
+      allowNull: false,
+      primaryKey: true,
+      defaultValue: DataTypes.UUIDV4,
+    },
+    user_id: {
+      type: DataTypes.UUID,
+      allowNull: false,
+    },
+    rapyd_e_wallet: {
```

Comment:
What do you think about moving these to a table: `rapyd`, the same as with the `user` mcs, where there is a `telegram` table?